deps/jerry-v8/src/v8jerry_utils.cpp

```diff
@@ -34,6 +34,37 @@ jerry_value_t JerryPolyfill::BuildMethod(const char* name, const char* fn_args,
     return method;
 }
 
+static jerry_value_t JerryHandlerStackTrace(const jerry_value_t func,
+                                            const jerry_value_t this_val,
+                                            const jerry_value_t args_p[],
+                                            const jerry_length_t args_count)
+{
+    if (!jerry_is_feature_enabled(JERRY_FEATURE_LINE_INFO))
+    {
+        printf("Line info disabled, no backtrace will be printed.\n");
```

Comment:
Shouldn't we return here if the line info is not supported?

Reply:
Yes, good idea. Done.